templates/new_device_login.txt

```diff
@@ -1,6 +1,8 @@
 {{t "New sign-in to Firefox" }}
 
 {{ device }}
+{{ ip }}
+{{ location }}
```

Comment:
A historical note: we created the original version of this email in a real hurry, and one of the constraints we worked under was that it not introduce any new strings.  Hence the raw undecorated device data being just plopped in the middle of the email like that.

We're no longer under that constraint, so perhaps @ryanfeeley can suggest something better.